src/redux/reducers/todosReducer.js

```diff
@@ -1,13 +1,31 @@
+import { ADD_TODO, IS_ADDING } from "redux/actions/actionTypes"
 export const initialState = {
-  todosItems: [
-    { text: "Eat", completed: true },
-    { text: "Sleep", completed: false },
-  ],
+  todoItems: [],
   visibilityFiler: "SHOW_ALL",
+  isAdding: false,
 }
 
 export default function todosReducer(state = initialState, action) {
   switch (action.type) {
+    case ADD_TODO:
+      const { text, createdAt, completed } = action.payload
```

Comment:
We should use `ID` which will be unique for each task, so that we don't face any problems identifying a specific task in the future steps like deleting a task.